js/sort_term_dropdowns.js

```diff
@@ -0,0 +1,85 @@
+
+function sortTermDropdown(){
+  var termSelect = $('select[name="enrollment_term_id"]');
+  var selectedOption = $(termSelect).val();
+  var termSelectList = $('option', termSelect);
+  var re_letter = /^[A-Za-z]/;
+  var re_number = /^[0-9]/;
+
+  var topList = new Array();
+  var middleList = new Array();
+  var bottomList = new Array();
+
+  for (var i=0; i < termSelectList.length; i++) {
+    if (termSelectList[i].label == 'All Terms') {
+      // this should be the first item in the list
+      topList.unshift(termSelectList[i]);
+    }
+    else if (termSelectList[i].label == 'Ongoing') {
+      // this should be the second item in the list
```

Comment:
Minor: Would it be safe to have a case insensitive match, although looks like Canvas has been using 'Ongoing' consistently.